docs/scheduler.rst

```diff
@@ -110,34 +110,9 @@ False``) or by default at the configuration file level with ``catchup_by_default
 will do, is to instruct the scheduler to only create a DAG Run for the most current instance of the DAG
 interval series.
 
-.. code:: python
-
-    """
-    Code that goes along with the Airflow tutorial located at:
-    https://github.com/apache/airflow/blob/master/airflow/example_dags/tutorial.py
-    """
-    from airflow import DAG
-    from airflow.operators.bash_operator import BashOperator
-    from datetime import datetime, timedelta
-
-
-    default_args = {
-        'owner': 'airflow',
-        'depends_on_past': False,
-        'start_date': datetime(2015, 12, 1),
-        'email': ['airflow@example.com'],
-        'email_on_failure': False,
-        'email_on_retry': False,
-        'retries': 1,
-        'retry_delay': timedelta(minutes=5)
-    }
-
-    dag = DAG(
-        'tutorial',
-        default_args=default_args,
-        description='A simple tutorial DAG',
-        schedule_interval='@daily',
-        catchup=False)
+.. literalinclude:: ../airflow/example_dags/tutorial.py
+    :language: python
+    :lines: 34-64
```

Comment:
Please use the option `:start-after:` and `:end-before:`. This makes documentation easier to maintain. If you are afraid of markers that appear in the content, if the fragments are overlapped, I have a ready fix in the queue. 
https://gitlab.polidea.com/Internal/Corgi/merge_requests/424
I am waiting for the right time to send it

Reply:
I cannot view the link. I guess it's an internal repo. 
Yes. I am afraid of markers showing in the content. 
I guess its not a big deal to show them though since they're just comments.

Reply:
Bad link. Good is available: https://gitlab.polidea.com/snippets/22